server/models/migrations/next/migrate.sql

```diff
@@ -73,4 +73,31 @@ DROP TABLE IF EXISTS `depot_distribution_permission`;
 CREATE TABLE `depot_distribution_permission` (
   `depot_uuid` BINARY(16) NOT NULL,
   `distribution_depot_uuid` BINARY(16) NOT NULL
-) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+/*
+ * @author: lomamech
+ * @date: 2020-10-05
+ * @desc: Parameter setting of the CMM calculation algorithm to be used #4984
+ */
+DROP TABLE IF EXISTS `average_consumption_algo`;
+
+CREATE TABLE `average_consumption_algo` (
+  `id` TINYINT(3) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(100) NOT NULL,
+  `label` VARCHAR(100) NOT NULL,
+  `comment` VARCHAR(100) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `average_consumption_algo` (`name`)
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+
+-- Average Consumption Algorithm
+INSERT INTO `average_consumption_algo` VALUES
+  (1, 'algo1', 'STOCK.MONTHLY_CONSUMPTION.ALGO_1', 'STOCK.MONTHLY_CONSUMPTION.ALGO_1_COMMENT'),
+  (2, 'algo2','STOCK.MONTHLY_CONSUMPTION.ALGO_2', 'STOCK.MONTHLY_CONSUMPTION.ALGO_2_COMMENT'),
+  (3, 'algo3', 'STOCK.MONTHLY_CONSUMPTION.ALGO_3', 'STOCK.MONTHLY_CONSUMPTION.ALGO_3_COMMENT'),
+  (4, 'algo_msh', 'STOCK.MONTHLY_CONSUMPTION.ALGO_4', 'STOCK.MONTHLY_CONSUMPTION.ALGO_4_COMMENT');
+
+ALTER TABLE `stock_setting` ADD COLUMN `average_consumption_algo_id` TINYINT(3) UNSIGNED NOT NULL DEFAULT 3;
```

Comment:
Awesome 👍  Thanks for adding these to the migration script.

Reply:
Sorry, I must update the migrate filte